modules/i18n/src/models/common.ts

```diff
@@ -44,15 +44,15 @@ export interface EditPage extends Page {
  * Model for notification set
  */
 export interface Notification {
-    error: NotificationItem;
-    genericError: NotificationItem;
-    success: NotificationItem;
+    error?: NotificationItem;
```

Comment:
Shall we keep these mandatory. If any error variation is not needed we can put `null`.

Reply:
This will be fixed in an upcoming PR.